demo/docker-compose.yml

```diff
@@ -104,14 +104,16 @@ services:
 
   apisix:
     container_name: apisix.example.com
-    image: apache/apisix:3.2.0-debian
+    image: linagora/apisix:3.2.0-debian-javaplugin
```

Comment:
IMO we should plan to let the CI build and publish the image

Reply:
Can we remove `jwt-revoker` container?

Reply:
Logically krakend too then :)